functions/.eslintrc.json

```diff
@@ -0,0 +1,123 @@
+{
+  "parserOptions": {
+    // Required for certain syntax usages
+    "ecmaVersion": 6
+  },
+  "plugins": [
+    "promise"
+  ],
+  "extends": "eslint:recommended",
```

Comment:
any thoughts on extending AirBNB-base (for more consistency with the front-end)?

Reply:
Good idea, will do